ov-nginx/nginx.conf

```diff
@@ -3,8 +3,20 @@ events {
 }
 
 http {
+
+  # Admin
+  server {
+    listen 80;
+    server_name ov-admin.k8s.wgbhdigital.org;
+    location / {
+      proxy_pass http://ov-wag;
+    }
+  }
+
+  # Frontend
   server {
     listen 80 default_server;
+    server_name ovfrontend.k8s.wgbhdigital.org;
```

Comment:
ov-frontend vs ovfrontend?

Reply:
Came configured that way. 

`ov-front`?

Reply:
ah, i see. i guess we can change it later if we want, but shouldn't block this.